apps/e2e-tests/tests/tree-widget.spec.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { test, expect } from '@playwright/test';
+import assert from "assert";
+import { expandStagePanel, locateNode, locateWidget } from "./utils";
+
+test.describe("tree-widget", () => {
+
+  test.beforeEach(async ({ page, baseURL }) => {
+    assert(baseURL);
+    await page.goto(baseURL);
+  })
+
+  test.describe("should match image snapshot", () => {
+
+    const testCases = () => {
+
+      test("initial tree", async ({ page }) => {
+        const treeWidget = locateWidget(page, "tree");
+        await treeWidget.waitFor();
+
+        // wait for element to be visible in the tree
+        await locateNode(treeWidget, "ProcessPhysicalModel").getByRole("checkbox", { name: "Visible", exact: true }).waitFor();
+        await expect(treeWidget).toHaveScreenshot();
+      })
+
+      test("expanded tree node", async ({ page }) => {
+        const treeWidget = locateWidget(page, "tree");
+        await treeWidget.waitFor();
+
+        const node = locateNode(treeWidget, "ProcessPhysicalModel");
+        await node.getByRole("button").click();
```

Comment:
could we be more specific about what we're clicking? nodes may have other buttons (e.g. custom action buttons, filtering button)